frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-details-page-content.component.html

```diff
@@ -1,12 +1,18 @@
-<app-common-section-card-list>
-  <app-feature-flag-overview-details-section-card section-card></app-feature-flag-overview-details-section-card>
-  <app-feature-flag-inclusions-section-card *ngIf="(activeTabIndex$ | async) === 0" section-card
-    >inclusions-card</app-feature-flag-inclusions-section-card
-  >
-  <app-feature-flag-exclusions-section-card *ngIf="(activeTabIndex$ | async) === 0" section-card
-    >exclusions-card</app-feature-flag-exclusions-section-card
-  >
-  <app-feature-flag-exposures-section-card *ngIf="(activeTabIndex$ | async) === 1" section-card
-    >exposures-card</app-feature-flag-exposures-section-card
-  >
-</app-common-section-card-list>
+<div class="content-container">
```

Comment:
if we load the sub to the feature flag directly with an async pipe, we can do this and eliminate half of the component code: 

```
<ng-container *ngIf="(featureFlag$ | async) as featureFlag; else loading">
  <app-common-section-card-list class="common-section-card-list">
    <app-feature-flag-overview-details-section-card [data]="featureFlag" section-card></app-feature-flag-overview-details-section-card>
    <app-feature-flag-inclusions-section-card [data]="featureFlag" *ngIf="(activeTabIndex$ | async) === 0" section-card>inclusions-card</app-feature-flag-inclusions-section-card>
    <app-feature-flag-exclusions-section-card [data]="featureFlag" *ngIf="(activeTabIndex$ | async) === 0" section-card>exclusions-card</app-feature-flag-exclusions-section-card>
    <app-feature-flag-exposures-section-card [data]="featureFlag" *ngIf="(activeTabIndex$ | async) === 1" section-card>exposures-card</app-feature-flag-exposures-section-card>
  </app-common-section-card-list>
</ng-container>

<ng-template #loading>
  <mat-spinner></mat-spinner>
</ng-template>
```

Reply:
then in component code, you just need this:

```typescript
featureFlag$ = this.featureFlagsService.selectedFeatureFlag$
```

Reply:
yeah i'll need this as an observable like suggested above in order to complete some work I'm currently doing, let me know if this suggestion works for you, I tried it in my local and it works fine.

Reply:
Yes works for me too